iOS/src/toga_iOS/libs/uikit.py

```diff
@@ -147,6 +147,12 @@ class UIInterfaceOrientation(Enum):
     LandscapeRight = 4
 
 
+class UIApplicationState(Enum):
+    UIApplicationStateActive = 0
+    UIApplicationStateInactive = 1
+    UIApplicationStateBackground = 2
```

Comment:
Where is this being used?

Reply:
Probably a stray content from previous changes. I have removed it.